### ISSUE TITLE
about compibility with INMP441

### ISSUE THREAD

**Issue by sbabadag** [MhageGH/esp32_CloudSpeech#4]
Hi ;
is the code compitible with INMP441 electret microphone or which changes we have to make ?
Thanks.

**Comment by dominickairmyne** [comment]
Did you ever give this a try? Wondering if it worked for you want to do the same thing!

**Comment by sbabadag** [comment]
Yes it worked , just after I wanted to investigate more precious microphone that has far field abilities , but did not go further
